CMakeLists.txt

```diff
@@ -17,7 +17,8 @@ function(embed_resources out_var)
     set(${out_var} "${result}" PARENT_SCOPE)
 endfunction()
 
-find_package(Boost COMPONENTS filesystem REQUIRED)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
`set(CMAKE_CXX_STANDARD 11)` would be a better choice. It is introduced in CMake 3.1. Since this project already requires CMake 3.5, it is safe to use.